src/icons/done.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
We have the Done svg in the file Administered.svg This would be a duplicate icon